lib/routes/client/key_data.js

```diff
@@ -0,0 +1,113 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const Joi = require('joi');
+
+const AppError = require('../../error');
+const db = require('../../db');
+const logger = require('../../logging')('routes.key_data');
+const P = require('../../promise');
+const validators = require('../../validators');
+const verify = require('../../browserid');
+
+const STATIC_EPOCH_ACCESS_KEY = Buffer.alloc(32).toString('hex');
```

Comment:
I'm gonna be honest, I don't like this new name that Brian suggested - reading it, I don't know what an "epoch" is supposed to be and it's not obvious how it will be used by client code.  I don't have an alternative that I really *like* though...

What do you think of "additional key material"?  It captures some of the essence of how we intend the client to use it (as an additional input to the key derivation function) without introducing any new concepts like "epoch" that might be mis-understood.

Anyway, I guess it's pretty easy for us to change the name of this thing later on since it's an internal implementation detail, so we don't need to block on it.